R/load.R

```diff
@@ -660,7 +660,11 @@ loadDistances <- function(input = "distances.csv", spatial) {
       dist.mat <- NULL
     }
   } else {
```

Comment:
what if we remove this "else," make dist.mat a matrix regardless of where it came from, and instead change the lines that can't handle a matrix input?

Reply:
so, line 662 would just end the if-statement.
Line 667 becomes `as.matrix` again.
And we change lines 687 and 688 from:
```r
      dist.mat[nrow(dist.mat) + 1, ] <- NA
      dist.mat[, ncol(dist.mat) + 1] <- NA
```
to:
```r
      dist.mat <- rbind(dist.mat, NA)
      dist.mat <- cbind(dist.mat, NA)
```

Then we wouldn't need the as.matrix in the end.

Reply:
Thank you for this @hugomflavio , 

I have pushed new code to incorporate what you provided in addition to some edits allowing successful passing of related checks to loadDistances(). 

Cheers,